.appveyor.yml

```diff
@@ -0,0 +1,69 @@
+# An AppVeyor build configuration file for the Barrier project
+
+
+##### General Configuiration #####
+version: '0.0.0.{APPVEYOR_REPO_COMMIT}.{build}'
+
+#environment:
+  # these variables are common to all jobs
+#  common_var1: value1
+#  common_var2: value2
+
+image:
+  - Visual Studio 2017
+  - Ubuntu
+
+platform:
+  - x86
```

Comment:
Apparently AppVeyor also offers x64, which I assume just means x86_64. If we could build 32-bit and 64-bit binaries, it'd be a plus.

Reply:
That should be possible.  I wanted to start out with a basic working set first.